frontends/PyCDE/integration_test/support/loopback.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/python3
```

Comment:
Can this and [loopback.py](https://github.com/llvm/circt/pull/3632/files#diff-ece9c4e3d5cf4d32c267b59f40030bb23e1c70125e6a0de78557e2c0a8990bda) be either merged or made more distinct? It looks to me like they're duplicating a fair amount, so it might be reasonable to factor out `LoopbackTester` to be shared.

Reply:
A lot of this was factored out into CosimBase, so I copied and customized (as you correctly surmised). I did miss removing the two tests which aren't run.